javascript/javascript3/week3/paintcircle/shoppingcart/shoppingcart.js

```diff
@@ -0,0 +1,156 @@
+//Getting into promises
+
+class respiratory {
+    constructor(username1, username2, username3) {
+        this.username1 = username1;
+        this.username2 = username2;
+        this.username3 = username3;
+
+        const promise1 = fetch(`https://api.github.com/search/repositories?q=user:${this.username1}`)
+        const promise2 = fetch(`https://api.github.com/search/repositories?q=user:${this.username2}`)
+        const promise3 = fetch(`https://api.github.com/search/repositories?q=user:${this.username3}`)
+        const allPromises = [promise1, promise2, promise3]
+        Promise.all([promise1, promise2, promise3])
+            .then(response => Promise.all([response[0].json(), response[1].json(), response[2].json()]))
+            .then(dat => dat.forEach(data => {
+
+                //const x =data.forEach(dat=>dat.full_name);
+                // const y =data.items[0].url;
+                //const z=data.items[0].owner.login;
+                const ul = document.createElement('ul');
+                document.body.appendChild(ul)
+                const h5 = document.createElement('h5');
+
+                const li1 = document.createElement('li');//
+                h5.innerHTML = `fullname of the repo - ${data.items[0].full_name}`;
+
+                li1.innerHTML = `url of the repo - ${data.items[0].url}`;
+
+                const li2 = document.createElement('li');
+                li2.innerHTML = `owner of the repo - ${data.items[0].owner.login}`;
+
+
+                ul.append(h5, li1, li2)
+            })
+
+            )
+
+    }
+}
+
+
+const use = new respiratory("Hailegebriel22", "Hanouj", "patel-prakashkumar")
+
```

Comment:
This const use(can call new respiratory without it) and all promise doesn't do anything and you can just delete them.

Reply:
I am creating instants using the three user names "Hailegebriel22", "Hanouj",  and "patel-prakashkumar"
   `class respiratory {
    constructor(username1, username2, username3) {..........
........}
....}`